### DISCUSSION TITLE
Add Mutually exclusive filter

### DISCUSSION THREAD

**Discussion by HenriPiot-dev** [codingjoe/django-select2#115] [Ideas]
Hello, I just start using this module and I found it very usefull ! For my use case I need to set mutually exclusive filter, imagine 2 select and one is fill by some values, when you want to fill the other on the first one is set to the empty state. what do you think about it ?

**Comment by codingjoe** [comment]
Hi @HenriPiot-dev, interesting idea. I believe the use case is a little narrow for most. However, you might already be able to do this. We have a dependent field feature. This might allow you to alter the related queryset in a way that will give you the desired result.
